kuma/wiki/tests/test_utils.py

```diff
@@ -0,0 +1,118 @@
+import os.path
+
+import mock
+from googleapiclient.http import HttpMockSequence
+from googleapiclient.errors import HttpError
+from kuma.core.tests import KumaTestCase
+
+from ..utils import analytics_user_counts
+
+
+class AnalyticsUserCountsTests(KumaTestCase):
+    def setUp(self):
```

Comment:
This might be a candidate for the [setUpClass](https://docs.python.org/2/library/unittest.html#setupclass-and-teardownclass) class method. This is also the first time I've ever said that.